src/Mod/Sketcher/App/planegcs/Constraints.h

```diff
@@ -155,6 +157,47 @@ namespace GCS
         double grad(double *) override;
     };
 
+    // Center of Gravity
```

Comment:
This one is indeed intended for future constraints. It's useful, for example, for a general centre of a triangle. Additionally, if we ever design a purely nonrational Bspline, this would be more optimal. I'm OK with removing it but I think there's no harm in keeping either.

Reply:
I do not mind keeping it, if it is unused code intended for future use. But if we keep it, we should review it (something I have not done in detail, will do now).